build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -45,6 +45,35 @@
       </ProjectsToTest>
     </ItemGroup>
 
+    <!-- Begin Workaround lack of a stable package version for depedencies; copy into simulated stable version folders -->
+ 
+    <PropertyGroup>
+        <StableSharedFrameworkDirectory>$(StageDirectory)/shared/Microsoft.NETCore.App/$(SharedFrameworkStableVersion)</StableSharedFrameworkDirectory>
+        <StableAdditionalSharedFrameworkDirectory>$(StageDirectory)/shared/Microsoft.NETCore.App/$(AdditionalSharedFrameworkStableVersion)</StableAdditionalSharedFrameworkDirectory>
+    </PropertyGroup>
+
+    <ItemGroup>  
+        <Stabilize_SourceFiles_1_1 Include="$(StageDirectory)/shared/Microsoft.NETCore.App/$(CLI_SharedFrameworkVersion)/*.*"/>  
+        <Stabilize_SourceFiles_1_0 Include="$(StageDirectory)/shared/Microsoft.NETCore.App/$(CLI_SharedFrameworkVersion_1_0)/*.*"/>  
+        <HackedSharedFrameworkDirectories Condition=" '$(IsSharedFrameworkPreReleaseVersion)' == 'True' "
+                                          Include="$(StableSharedFrameworkDirectory)" />
+        <HackedSharedFrameworkDirectories Condition=" '$(IsAdditionalSharedFrameworkPreReleaseVersion)' == 'True' "
+                                          Include="$(StableAdditionalSharedFrameworkDirectory)" />
```

Comment:
I prefer to avoid Hacked in source code. If this is how we want to do things permanently, we should name it accordingly. SimulatedStableSharedFrameworkDirectories or something.

Reply:
Sounds good to me. I wasn't entirely happy with that either.